### ISSUE TITLE
Option to remove whitspace around autocompleted import statements

### ISSUE THREAD

**Issue by silverwind** [sublimelsp/LSP-typescript#166]
Is there an option to change space around variables in a autocompleted `import` statement? e.g.

```js
import { something } from "./something.js";
```

into

```js
import {something} from "./something.js";
```

I tried disabling space-related settings but none seem to have any effect on import statements.

**Comment by rchl** [comment]
Not aware of a settings for that.

Your options are to create a feature request in https://github.com/microsoft/TypeScript/issues or use LSP-eslint with specific rule that correct that on save.

**Comment by silverwind** [comment]
Ok, assuming typescript has this option, do you need to add support here or is that automatic?

**Comment by rchl** [comment]
If it doesn't work after setting this one then try formatting the document first (LSP: Format file) and then try completions again. If it works then then it's something that should be fixed in the server.

**Comment by rchl** [comment]
Tested this myself and noticed that depending on how the import is completed, it either follows or doesn't follow the settings:

https://user-images.githubusercontent.com/153197/192781891-75440cef-0463-4b23-86bd-cd609fdcfb90.mov

I've reproduced it in VSCode also so I would suggest just reporting it as a bug in TypeScript.

If you report it against VSCode then it has much higher chance of being looked at. It's enough to set `  "typescript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": false` to reproduce there.

**Comment by silverwind** [comment]
I can reproduce in ST as well, `import foo<tab>`, looks to just ignore the `insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces` option.

**Comment by rchl** [comment]
Yes, I know it reproduces but since both TypeScript and VSCode are made by Microsoft it's easier for them to look at reports that report against VSCode.

**Comment by silverwind** [comment]
I looked into this again. Behaviour is the same in VSCode, so it's definitely not a bug in LSP.

Found this old issue about it: https://github.com/microsoft/TypeScript/issues/9613
